src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1190,6 +1190,25 @@ private Symbol UnwrapAlias(Symbol symbol, out AliasSymbol alias, BindingDiagnost
             }
 
             return TypeWithAnnotations.Create(AreNullableAnnotationsEnabled(node.TypeArgumentList.GreaterThanToken), resultType);
+
+            // Determines whether node represents containing type of an attribute.
+            // For example, returns true for "A<T>" node in "[A<T>.B]".
+            static bool ContainsAttributeType(NameSyntax node)
```

Comment:
>C [](http://example.com/codeflow?start=24&length=1)

We typically use lowercase for the first letter of local function names.

Reply:
5e7ac70c223fd907058bc07ef8773819e58d5da3